<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyze3D</class>
 <widget class="QWidget" name="Analyze3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyGLWidget" name="myGLWidget" native="true">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>60</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="xRotSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="yRotSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="zRotSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>xRot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>yRot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>zRot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>MyGLWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
